internal/controller/flp/flp_controller.go

```diff
@@ -232,59 +233,94 @@ func reconcileMonitoringCerts(ctx context.Context, info *reconcilers.Common, tls
 }
 
 func (r *Reconciler) getOpenShiftSubnets(ctx context.Context) ([]flowslatest.SubnetLabel, error) {
-	var subnets []flowslatest.SubnetLabel
+	if !r.mgr.ClusterInfo.HasCNO() {
+		return nil, nil
+	}
+	var pods, services, machines, extIPs []string
 
 	// Pods and Services subnets are found in CNO config
-	if r.mgr.ClusterInfo.HasCNO() {
-		network := &configv1.Network{}
-		err := r.Get(ctx, types.NamespacedName{Name: "cluster"}, network)
-		if err != nil {
-			return nil, fmt.Errorf("can't get Network information: %w", err)
-		}
-		var podCIDRs []string
-		for _, podsNet := range network.Spec.ClusterNetwork {
-			podCIDRs = append(podCIDRs, podsNet.CIDR)
-		}
-		if len(podCIDRs) > 0 {
-			subnets = append(subnets, flowslatest.SubnetLabel{
-				Name:  "Pods",
-				CIDRs: podCIDRs,
-			})
-		}
-		if len(network.Spec.ServiceNetwork) > 0 {
-			subnets = append(subnets, flowslatest.SubnetLabel{
-				Name:  "Services",
-				CIDRs: network.Spec.ServiceNetwork,
-			})
-		}
-		if network.Spec.ExternalIP != nil && len(network.Spec.ExternalIP.AutoAssignCIDRs) > 0 {
-			subnets = append(subnets, flowslatest.SubnetLabel{
-				Name:  "ExternalIP",
-				CIDRs: network.Spec.ExternalIP.AutoAssignCIDRs,
-			})
-		}
+	network := &configv1.Network{}
+	err := r.Get(ctx, types.NamespacedName{Name: "cluster"}, network)
+	if err != nil {
+		return nil, fmt.Errorf("can't get Network (config) information: %w", err)
+	}
+	for _, podsNet := range network.Spec.ClusterNetwork {
+		pods = append(pods, podsNet.CIDR)
+	}
+	services = network.Spec.ServiceNetwork
+	// API server
+	// TODO: get IP from EndpointSlice kubernetes
+	services = append(services, "172.20.0.1/32")
+	if network.Spec.ExternalIP != nil && len(network.Spec.ExternalIP.AutoAssignCIDRs) > 0 {
+		extIPs = network.Spec.ExternalIP.AutoAssignCIDRs
 	}
 
 	// Nodes subnet found in CM cluster-config-v1 (kube-system)
 	cm := &corev1.ConfigMap{}
 	if err := r.Get(ctx, types.NamespacedName{Name: "cluster-config-v1", Namespace: "kube-system"}, cm); err != nil {
 		return nil, fmt.Errorf(`can't read "cluster-config-v1" ConfigMap: %w`, err)
 	}
-	machines, err := readMachineNetworks(cm)
+	// Machines
+	machines, err = readMachineFromConfig(cm)
 	if err != nil {
 		return nil, err
 	}
 
-	if len(machines) > 0 {
-		subnets = append(subnets, machines...)
+	// Additional OVN subnets
+	networkOp := &operatorv1.Network{}
+	err = r.Get(ctx, types.NamespacedName{Name: "cluster"}, network)
```

Comment:
```suggestion
	err = r.Get(ctx, types.NamespacedName{Name: "cluster"}, networkOp)
```
If I understand correctly this is networkOp that you wanted to get here.